operator/hack/addons.yaml

```diff
@@ -88,55 +153,52 @@ spec:
             - mountPath: /var/log/journal
               name: journal
               readOnly: true
-      serviceAccountName: loki-promtail
-      tolerations:
-        - effect: NoSchedule
-          key: node-role.kubernetes.io/master
+          securityContext:
+            readOnlyRootFilesystem: true
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop:
+                - ALL
+      serviceAccountName: promtail-loki
       volumes:
         - configMap:
             defaultMode: 420
-            name: loki-promtail
+            name: promtail-loki
           name: config
-        - hostPath:
-            path: /run/promtail
-            type: ""
+        - emptyDir: {}
           name: run
-        - hostPath:
-            path: /var/lib/docker/containers
-            type: ""
+        - emptyDir: {}
           name: docker
-        - hostPath:
-            path: /var/log/pods
-            type: ""
+        - emptyDir: {}
           name: pods
-        - hostPath:
-            path: /var/log/journal
-            type: ""
+        - emptyDir: {}
           name: journal
-
 ---
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: loki-promtail
-  namespace: loki
+  name: promtail-loki
   labels:
     app.kubernetes.io/name: promtail
 data:
   promtail.yaml: |
-    client:
-      backoff_config:
-        min_period: 100ms
-        max_period: 5s
-        max_retries: 5
-      batchsize: 102400
-      batchwait: 1s
-      external_labels: {}
-      timeout: 10s
+    clients:
+      - url: http://lokistack-gateway-http-lokistack-dev.openshift-logging.svc:8080/api/logs/v1/application/loki/api/v1/push
```

Comment:
With #5607 the new svc is `lokistack-dev-gateway-http` instead of `lokistack-gateway-http-lokistack-dev`